extensions/applicationinsights-properties-js/src/Context/User.ts

```diff
@@ -61,10 +61,14 @@ export class User implements IUserContext {
     constructor(config: ITelemetryConfig, core: IAppInsightsCore) {
         let _logger = safeGetLogger(core);
         let _cookieManager: ICookieMgr = safeGetCookieMgr(core);
+        let _storageNamePrefix: () => string;
 
         dynamicProto(User, this, (_self) => {
+            _self.config = config;
+            _storageNamePrefix = () => _self.config.namePrefix && _self.config.namePrefix() ? User.userCookieName + _self.config.namePrefix() : User.userCookieName;
```

Comment:
As mentioned above, we should not just start applying this for everyone.

Reply:
Thanks for the suggestions Nev. I updated the logic as below: 

- Add `sessionCookiePostfix` and `userCookiePostfix`.

For session cookie:
- if both `sessionCookiePostfix` and `namePrefix` is configured, then ignore `namePrefix` and use `sessionCookiePostfix`.
- if `sessionCookiePostfix` not configured but `namePrefix` is configured, then use `namePrefix`. 
- if none is configured, no suffix is added.

For user cookie: 
- `namePrefix` has no effect on user cookie.
- if `userCookiePostfix` is configured, append `userCookiePostfix` to `ai_user` cookie name.
- if none is configured, no suffix is added.